### ISSUE TITLE
Review accessibility

### ISSUE THREAD

**Issue by bjackson27** [google/site-kit-wp#173]
## Feature Description

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* Accessibility of all Site Kit interfaces and elements added to non-owned screens should be audited
* New issues should be opened to remediate issues identified by this review

## Implementation Brief

* N/A

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by aaemnnosttv** [comment]
@tofumatt this is an old issue but one worth revisiting soon I think since we're over the mountain of refactoring since the initial release.

Given your expertise in the area it would be good for you to take the lead in defining how we should approach this.

**Comment by FlicHollis** [comment]
Hi @tofumatt it would be great if we could get this defined and moved to IB, please could you let me know if you have time or alternatively if there is anyone else you would recommend to pick this up? Thanks!

**Comment by aaemnnosttv** [comment]
Updated AC here but this is one ultimately for @tofumatt as our resident a11y specialist.

@tofumatt let me know if this sounds good. We should probably use the estimate as a time box for the review but feel free to add some structure as an IB. Back to you 👍

**Comment by bethanylang** [comment]
@tofumatt Circling back on this one, please and thank you!

**Comment by tofumatt** [comment]
Accessibility issues from the above document are all filed with ACs ready:

* #6635
* #6636
* #6637
* #6638
* #6641
* #6642

That's all that's needed from this issue, as it was about reviewing/auditing the plugin. Work on accessibility fixes can be done in the issues above, so closing this issue. 👍🏻
